developer/js/source/model-info-compiler/model-info-compiler.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * Merges a source .model_info file with metadata extracted from .kps file and
+ * compiled files to produce a comprehensive .model_info file.
+ */
+
+/// <reference path="../lexical-model-compiler/lexical-model.ts" />
+/// <reference path="../lexical-model-compiler/model-info-file.ts" />
+
+import * as fs from "fs";
+import * as path from "path";
+
+const minKeymanVersion = '12.0';
+
+export default class ModelInfoCompiler {
```

Comment:
What is the purpose of this class, with no constructor, no private or public properties, and no reference to `this`? Perhaps this is better expressed as a simple function?